apps/explorer/src/ui/InternalLink.tsx

```diff
@@ -22,7 +22,7 @@ export function AddressLink({ address, noTruncate }: AddressLinkProps) {
         ? address
         : truncate(address, TRUNCATE_LENGTH);
     return (
-        <Link variant="mono" to={`/address/${encodeURIComponent(address)}`}>
+        <Link variant="mono" to={`/addresses/${encodeURIComponent(address)}`}>
```

Comment:
I put up a PR to propose we actually do singular routes: https://github.com/MystenLabs/sui/pull/6424

Reply:
revert this back